research/picai/central/README.md

```diff
@@ -0,0 +1,14 @@
+# Running Centralized Example
+
+The following instructions outline training and validating a simple U-Net model on the Preprocessed Dataset described in the [PICAI Documentation](/research/picai/README.md) using a centralized setup. An example of the usage is below. Note that the script needs to be run from the top level of the FL4Health repository. Moreover, a python environment with the required libraries must already exist.  See the main PICAI documentation Cluster [PICAI Documentation](/research/picai/README.md) for instructions on creating and activating environment required to exectute the following code. The commands below should be run from the top level directory:
+
+```bash
+./research/picai/central/launch.slrm folder_for_artifacts/ path_to_desired_venv/ fold_id
```

Comment:
Super minor: since we are passing run_name now, should we include it in the example here?